dev-docs/bidders/smartxsp.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: bidder
+title: SmartXSP
+description: Prebid SmartXSP Bidder Adaptor
+pbjs: true
+biddercode: smartxsp
+media_types: banner
+sidebarType: 1
```

Comment:
Can you add more meta data to make it easier for reads to vet your adapter.
Full list can be found here: https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

```
gdpr_supported: true/false
gvl_id: none
usp_supported: true/false
coppa_supported: true/false
gpp_supported: true/false
safeframes_ok: true/false
```

Reply:
make that tcfeu_supported instead of gdpr_supported. And for the record, the value must be 'false' unless a GVL ID is supplied